ci_config.json

```diff
@@ -5,4 +5,9 @@
       "WITH_ZSTD=disabled"
     ]
   }
+  "sfml": {
```

Comment:
Oops, typo.

Reply:
I hate that JSON does not allow coma on last item of the list.

Reply:
It is pretty bizarre, yes...

I guess there is always ini. :)